libs/python.json

```diff
@@ -128,5 +128,7 @@
   {"id":"py.gflags","imports":["gflags"],"name":"gflags","repo":"google/python-gflags","tags":["argument-parsing"],"tech":["utils"],"status":"model-ready"},
   {"id":"py.gooey","imports":["gooey"],"name":"gooey","repo":"chriskiehl/Gooey","tags":["cli2gui"],"tech":["utils"],"status":"model-ready"},
   {"id":"py.psutil","imports":["psutil"],"name":"psutil","repo":"giampaolo/psutil","tags":["system"],"tech":["utils"],"status":"awaiting-model"},
-  {"id":"py.python-dateutil","imports":["dateutil"],"name":"dateutil","repo":"dateutil/dateutil","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"}
+  {"id":"py.python-dateutil","imports":["dateutil"],"name":"dateutil","repo":"dateutil/dateutil","tags":["date-time"],"tech":["utils"],"status":"awaiting-model"},
+  {"id":"py.dynaconf","imports":["dynaconf"],"name":"dynaconf","repo":"rochacbruno/dynaconf","tags":["configuration"],"tech":["utils"],"status":"model-ready"},
+  {"id":"py.flasgger","imports":["flasgger"],"name":"flasgger","repo":"rochacbruno/flasgger","tags":["micro-framework"],"tech":["python-web"],"status":"model-ready"}
```

Comment:
It's your repo so you know better, but just as a suggestion does it make sense to tag it as 'flask-openapi'? Tag is what shows under the library name, so it's nice to keep it as informative as possible.

<img width="447" alt="image" src="https://user-images.githubusercontent.com/20287615/54319602-5803a680-45a7-11e9-9082-875d33c66e7e.png">